flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/table/descriptors/KafkaValidator.java

```diff
@@ -36,6 +36,7 @@
 	public static final String CONNECTOR_VERSION_VALUE_09 = "0.9";
 	public static final String CONNECTOR_VERSION_VALUE_010 = "0.10";
 	public static final String CONNECTOR_VERSION_VALUE_011 = "0.11";
+	public static final String CONNECTOR_VERSION_VALUE_20 = "2.0";
```

Comment:
There is only residual information about kafka 2.0 here.

Reply:
@aljoscha It's here. The kafka version will be verified in KafkaValidator#validateVersion. I recommend removing the validation method and refactoring this constant as:

```java
public static final String CONNECTOR_UNTRACKED_VERSION_VALUE = "untracked"
```

What do you think?

Reply:
I think it's okay to leave it in for now because I don't want to mess with how the table API source/sink validation works for now.

Reply:
OK, agree.